### PR TITLE
Implement coord2address and category search

### PR DESCRIPTION
**Pull request by GoRyne**

### CODE REVIEW

File: v2/local/category_search.go [hunk 1]
```diff
@@ -1,2 +1,167 @@
...
+	X                 string
+	Y                 string
+	Radius            int
+	Rect              string
+	Page              int
+	Size              int
+	Sort              string
+}
+
+func CategorySearch(category_group_code string) *CategorySearchIterator {
```
Comment (maengsanha) [inline v2/local/category_search.go]: It would be better to `PlaceSearchByCategory(groupcode string)`.

File: v2/local/category_search.go [hunk 2]
```diff
@@ -1,2 +1,167 @@
...
+	c.Format = "xml"
+	return c
+}
+
+func (c *CategorySearchIterator) AuthorizeWith(key string) *CategorySearchIterator {
+	c.AuthKey = "KakaoAK " + strings.TrimSpace(key)
+	return c
+}
+
+func (c *CategorySearchIterator) SetLongitude(x string) *CategorySearchIterator {
```
Comment (maengsanha) [inline v2/local/category_search.go]: It would be better to `WithRadius(x, y float64, radius int)`.

File: v2/local/category_search.go [hunk 3]
```diff
@@ -1,2 +1,167 @@
...
+}
+
+func (c *CategorySearchIterator) SetRadius(radius int) *CategorySearchIterator {
+	if 0 <= radius && radius <= 20000 {
+		c.Radius = radius
+	}
+	return c
+}
+
+func (c *CategorySearchIterator) SetRect(rect string) *CategorySearchIterator {
```
Comment (maengsanha) [inline v2/local/category_search.go]: It would be better to `WithRect(rect int)`.

File: v2/local/category_search.go [hunk 4]
```diff
@@ -1,2 +1,167 @@
...
+}
+
+func (c *CategorySearchIterator) Display(size int) *CategorySearchIterator {
+	if 1 <= size && size <= 15 {
+		c.Size = size
+	}
+	return c
+}
+
+func (c *CategorySearchIterator) SortType(sort string) *CategorySearchIterator {
```
Comment (maengsanha) [inline v2/local/category_search.go]: It would be better to `SortBy(typ string)`.

File: v2/local/coord2address.go [hunk 5]
```diff
@@ -1,2 +1,138 @@
...
+	b.Format = "xml"
+	return b
+}
+
+func (b *CoordToAddressInitializer) AuthorizeWith(key string) *CoordToAddressInitializer {
+	b.AuthKey = "KakaoAK " + strings.TrimSpace(key)
+	return b
+}
+
+func (b *CoordToAddressInitializer) RequestWGS84() *CoordToAddressInitializer {
```
Comment (maengsanha) [inline v2/local/coord2address.go]: Organize `Request...` to `Input(coord string)`.

**Review by maengsanha — CHANGES_REQUESTED** [review]

**Comment by GoRyne** [comment]
Resolve all conversations please to check code

**Review by maengsanha — APPROVED** [review]

**Pull request merged**